docs/UserGuide.md

```diff
@@ -170,48 +162,65 @@ Format: `add-doctor n/NAME [t/TAG]…​`
 </div><br>
 
 Examples:
-
 * `add-doctor n/Dr Meredith Grey`
-
 * `add-doctor n/Dr Strange t/Dormammu t/IveComeToBargain`
 
 
-### Cleaning all entries in doctor records: `clear-doctor`
+### Clearing all entries in doctor records: `clear-doctor`
 Clears all entries from the doctor records. <br>
-If any appointments exist in the appointment schedule, then this command will not work. 
-`clear-appt` will have to be done first to clear the appointment schedule.
+* Entries in the doctor records can only be cleared if there are no existing appointments in the appointment schedule.
+* Otherwise, `clear-appt` have to called first.
 
 Format: `clear-doctor`
 
 ### Deleting a doctor : `delete-doctor`
-Deletes the specified doctor from the doctor records.
+Deletes the specified doctor from the doctor records.<br>
 
-Format: `delete-doctor INDEX`
+Format: `delete-doctor [--force] INDEX`
+
+* Deletes the doctor at the specified `INDEX`.
+* The index refers to the index number shown in the displayed doctor records.
+* The index **must be a positive integer** 1, 2, 3, …​
+* The specified doctor can only be deleted if there are no existing appointments associated with him/her in the appointment schedule.
+Otherwise, `--force` must be included to force delete the specified doctor, along with all associated appointments in the appointment schedule. 
+
+Examples:
+* `list-doctor` followed by `delete-doctor 3` deletes the 3rd doctor in the doctor records.
+* `list-doctor` followed by 'delete-doctor --force 1' force deletes the 1st doctor in the doctor records, along with all associated appointments in the appointment schedule.
+* `find-doctor Drake` followed by `delete-doctor 1` deletes the 1st doctor in the results of the `find-doctor` command.
 
-Similar to `delete-patient`
 
 ### Editing a doctor : `edit-doctor`
 Edits an existing doctor in the doctor records.<br>
 
 Format: `edit-doctor INDEX [n/NAME] [t/TAG]…​`
 
-* Edits the doctor at the specified INDEX. The index refers to the index number shown in the displayed doctor records. The index must be a <strong>positive integer</strong> 1, 2, 3, …​<br>
-
-Similar to `edit-patient`, except that there are less fields that are editable.
+* Edits the doctor at the specified `INDEX`. The index refers to the index number shown in the displayed doctor records. The index must be a **positive integer** 1, 2, 3, …​<br>
```

Comment:
same for this regarding repetitions, but perhaps more acceptable here since its different from edit-patient regarding the parameters